### DISCUSSION TITLE
Read source code which version is "v0.27.2" and then  have found some unnecessary code

### DISCUSSION THREAD

**Discussion by Hitotsubashi** [axios/axios#4919] [General]
I write a review [here](https://github.com/axios/axios/pull/3305/commits/37873c764418f8c0235c0702ca7a320bb0f90044#diff-1e23d795737c808b64379d393d916785d052ae1326fec630ed5285d77c8f6d48R52), I want to know are they necessary?

**Comment by jasonsaayman** [comment]
Please put in a PR so that the code may be removed
